src/tests/ftest/container/boundary.py

```diff
@@ -22,120 +22,138 @@ class BoundaryTest(TestWithServers):
+        self.io_obj_classs = self.params.get("obj_classs", '/run/container/execute_io/*')
+
+    def get_pool(self, *args, **kwargs):
+        """Get a test pool object and append to list.
+
+        Returns:
+            TestPool: the created test pool object.
+
+        """
+        pool = super().get_pool(*args, **kwargs)
+        self.pool.append(pool)
+        return pool
+
+    def create_container_and_test(self, pool=None, cont_num=1):
         """To create single container on pool.
+
         Args:
             pool (str): pool handle to create container.
-            pool_num (int): pool number.
             container_num (int): container number to create.
-            with_io (bool): enable container test with execute_io.
-        """
 
+        """
         try:
             container = self.get_container(pool)
         except (DaosTestError, TestFail) as err:
-            msg = "#(3.{}.{}) container create failed. err={}".format(pool_num, cont_num, err)
-            self.fail(msg)
+            self.fail(
+                "#(3.{}.{}) container create failed. err={}".format(pool.label, cont_num, err))
 
-        self.log.info("===(3.%d.%d)create_container_and_test, container %s created..",
-            pool_num, cont_num, container)
-        if with_io:
-            io_run_time = self.params.get("run_time", '/run/container/execute_io/*')
-            rank = self.params.get("rank", '/run/container/execute_io/*')
-            obj_classs = self.params.get("obj_classs", '/run/container/execute_io/*')
+        if self.with_io:
             try:
-                data_bytes = container.execute_io(io_run_time, rank, obj_classs)
-                self.log.info(
-                    "===(3.%d.%d)Wrote %d bytes to container %s", pool_num, cont_num,
-                    data_bytes, container)
+                data_bytes = container.execute_io(
+                    self.io_run_time, self.io_rank, self.io_obj_classs)
             except (DaosTestError, TestFail) as err:
-                msg = "#(3.{}.{}) container IO failed, err: {}".format(pool_num, cont_num, err)
-                self.fail(msg)
-        time.sleep(2)  #to syncup containers before close
+                self.fail(
+                    "#(3.{}.{}) container IO failed, err: {}".format(pool.label, cont_num, err))
+        time.sleep(2) # to syncup containers before close
 
         try:
-            self.log.info("===(4.%d.%d)create_container_and_test, container closing.",
-                pool_num, cont_num)
             container.close()
-            self.log.info("===(4.%d.%d)create_container_and_test, container closed.",
-                pool_num, cont_num)
         except (DaosTestError, TestFail) as err:
-            msg = "#(4.{}.{}) container close fail, err: {}".format(pool_num, cont_num, err)
-            self.fail(msg)
+            self.fail(
+                "#(4.{}.{}) container close failed, err: {}".format(pool.label, cont_num, err))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string